src/target/riscv/riscv.c

```diff
@@ -1933,6 +2085,13 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.usage = "riscv set_ir_idcode [idcode|dtmcs|dmi] value",
 		.help = "Set IR value for specified JTAG register."
 	},
+	{
+		.name = "use_bscan_tunnel",
```

Comment:
Can you also document this command in doc/openocd.texi?

Reply:
Yes, I've added, and viewed the resulting PDF as cross-check.